v01-object-or-null/src/main/java/dev/futa/tutorial/pesel/v01/PeselInfo.java

```diff
@@ -0,0 +1,29 @@
+package dev.futa.tutorial.pesel.v01;
+
+import dev.futa.tutorial.pesel.Gender;
+
+import java.time.LocalDate;
+
+public class PeselInfo {
+    private final String pesel;
+    private final LocalDate birthDate;
+    private final Gender gender;
+
+    PeselInfo(String pesel, LocalDate birthDate, Gender gender) {
+        this.pesel = pesel;
+        this.birthDate = birthDate;
+        this.gender = gender;
+    }
+
+    String getPesel() {
```

Comment:
Specjalnie package private?

Reply:
W zasadzie i tak na zewnątrz nie będzie użyte. Bo to demo. nie wiem jak podejść.

Reply:
Jeśli `PeselService` jest `public` oraz ma publiczne metody, które sugerują, że może być wszędzie wykorzystane, to analogicznie powinno być z `PeselInfo`.